src/sentry/static/sentry/app/components/activity/note.jsx

```diff
@@ -6,7 +6,52 @@ import ConfigStore from '../../stores/configStore';
 import LinkWithConfirmation from '../../components/linkWithConfirmation';
 import {t} from '../../locale';
 
+function isSafeHref(href) {
+  try {
+    return /^https?:\/\/./i.test(decodeURIComponent(unescape(href)));
+  } catch(e) {
+    return false;
+  }
+}
+
+
+// We need to implement our own marked Renderer to not render
+// potentially malicious uris.
+// This is copy/pasted from
+// https://github.com/chjj/marked/blob/master/lib/marked.js#L869-L888
+// and modified.
+function Renderer(options) {
+  this.options = options || {};
+}
+
+Object.assign(Renderer.prototype, marked.Renderer.prototype);
+
+Renderer.prototype.link = function(href, title, text) {
+  // For a bad link, just return the plain text href
+  if (this.options.sanitize && !isSafeHref(href)) return href;
```

Comment:
> Just use the absolute url. No need to use relative links.
 
Okay, that's easy to tell me, but it might confuse somebody pretty easily since you're not going to write a how-to anywhere.

> It's just plain text, it doesn't get rendered as DOM or anything, so yes. And since this is making things more restrictive, I'd rather opt to returning the original value rather than silently removing it so people are less confused.

Then do this. Wrap link, and if the output is `''`, output `href` instead. Again, I think this can be implemented with much less code and be more compatible with Marked.